test/app_web/api_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule AppWeb.APITest do
```

Comment:
The [`/test/app_web/api_test.exs`](https://github.com/dwyl/imgup/blob/eacf8543259403f4fd2f92e6b5ad50eca602e368/test/app_web/api_test.exs) file only contains tests where the status code is **`400`**. 🔴 
How is the "**_happy_ path**" being tested? 🤷‍♂️

Reply:
Happy path tested in: https://github.com/dwyl/imgup/blob/2f50de8bdfb22c6ae298256bcc39ea2e06a07cef/test/app/upload_test.exs